changelogs/unreleased/gh-6063-net-box-ping-args.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/Netbox
```

Comment:
We don't have separate subsystem "Netbox" in our release notes. See the possible options in `tools/gen-release-notes` script. Here it would be `bugfix/core`.

Reply:
Got it. Just seen that option in the previous release)